pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -742,7 +742,17 @@ protected void internalDeletePartitionedTopic(AsyncResponse asyncResponse,
                         .thenCompose(partitionedMeta -> {
                             final int numPartitions = partitionedMeta.partitions;
                             if (numPartitions < 1) {
-                                return CompletableFuture.completedFuture(null);
+                                return pulsar().getNamespaceService().checkNonPartitionedTopicExists(topicName)
+                                        .thenApply(exists -> {
+                                    if (exists) {
+                                        throw new RestException(Response.Status.CONFLICT,
+                                                String.format("%s is a non-partitioned topic. Instead of calling"
```

Comment:
Does `Instead of calling delete-partitioned-topic please call delete.` make sense?

Reply:
This prompt looks like it only applies to pulsar-admin, not to other clients.

Reply:
It is already `Instead of calling delete-partitioned-topic please call delete.`

Reply:
> It is already `Instead of calling delete-partitioned-topic please call delete.`

This is your code, I didn't change this.

`delete-partitioned-topic` is a pulsar-admin's command, this prompt is not accurate enough, and only applies to pulsar-admin's command, not to other clients.